.github/workflows/eas-update.yaml

```
@@ -0,0 +1,28 @@
+  pull_request:
+    types: [opened, synchronize, labeled]
+    branches:
+      - main
+
+jobs:
+  ota-update:
+    if: contains(github.event.pull_request.labels.*.name, 'eas-update')
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🏗 Setup repo
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)